src/routes/tweets.route.ts

```diff
@@ -0,0 +1,42 @@
+import express, { Request, Response } from "express";
+
+import * as tweetsService from "../services/tweets.sevice";
+
+export const tweetsRouter = express.Router();
+tweetsRouter.get("/:id", async (req: Request, res: Response) => {
+  const tweet = await tweetsService.getTweetById(req.params.id);
+  res.json(tweet);
+});
+
+tweetsRouter.get("/", async (req: Request, res: Response) => {
+  const tweets = await tweetsService.getAllTweets();
+  res.json(tweets);
```

Comment:
```suggestion
  res.send(tweets);
```
json is used under the hood. 
In other modules we write like this